src/routes/Home/components/HomePage/HomePage.js

```diff
@@ -39,7 +39,8 @@ class HomePage extends React.PureComponent {
     const { catalogs, t } = this.props
 
     return (
-      <DocumentTitle title={t('HomePage.home')}>
+      <div>
+        <Helmet title={t('HomePage.home')} />
```

Comment:
If we have a title+default title in <App /> we don’t need a specific title in the home page.